kolibri/plugins/learn/templates/learn/learn.html

```diff
@@ -0,0 +1,11 @@
+{% extends "kolibri/base.html" %}
+{% load learn_tags kolibri_tags %}
+{% block frontend_assets %}
+{{ block.super }}
+{% frontend_assets 'kolibri.plugins.learn.KolibriLearnFrontEnd' %}
+{% content_renderer_frontend_async %}
```

Comment:
With synchronous loading, I think it makes sense to have different tags, like `xyz_sync` for loading a set of resources and `frontend_assets` for loading an individual resource. But for async loading, why have multiple async tags? Do you think it makes sense to have *all* modules included with e.g. the `base_async` tag? That way *any* module could in principle be loaded async with no extra effort, whereas here I have to keep track of which modules are registered to `content_renderer_frontend_async` and also `base_async`, as well as manually registering my async modules through the plugin system.